File: test/fixtures/output/javascript/fetch/application-form-encoded.js
const options = {
  method: 'POST',
  headers: {'content-type': 'application/x-www-form-urlencoded'},
  body: new URLSearchParams({foo: 'bar', hello: 'world'})
};

fetch('https://httpbin.org/anything', options)
  .then(response => response.json())
  .then(response => console.log(response))
  .catch(err => console.error(err));

File: test/fixtures/output/r/httr/application-form-encoded.r
library(httr)

url <- "https://httpbin.org/anything"

payload <- "foo=bar&hello=world"

encode <- "form"

response <- VERB("POST", url, body = payload, content_type("application/x-www-form-urlencoded"), encode = encode)

content(response, "text")
